### PR TITLE
docs: run local `--proxy` blog

### PR DESCRIPTION
**Pull request by dannyrandall**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the Apache 2.0 License.

### CODE REVIEW

File: site/content/blogs/release-v132.en.md [hunk 1]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
 
     See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
-## 
+## `copilot run local --proxy`
+
+The new `--proxy` flag on `copilot run local` enhances your local development experience by making to possible for your local containers to talk to services deployed in your environment's VPC.
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
The new `--proxy` flag on `copilot run local` enhances your local development experience by making it possible for your local containers to talk to services deployed in your environment's VPC.
```

File: site/content/blogs/release-v132.en.md [hunk 2]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
...
+
+For example, imagine you have an environment with two services, `foo` and `bar`, who both have [Service Connect](TODO) enabled.
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
For example, imagine you have an environment with two services, `foo` and `bar`, who both have [Service Connect](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/service-connect.html) enabled.
```

File: site/content/blogs/release-v132.en.md [hunk 3]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
...
+Additionally, `foo` has an [RDS addon](TODO) deployed that it uses for storing data.
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
Additionally, `foo` has an [RDS addon](https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2.html) deployed that it uses for storing data.
```

File: site/content/blogs/release-v132.en.md [hunk 4]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
 
     See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
-## 
+## `copilot run local --proxy`
+
+The new `--proxy` flag on `copilot run local` enhances your local development experience by making it possible for your local containers to talk to services deployed in your environment's VPC.
+
+For example, imagine you have an environment with two services, `foo` and `bar`, who both have [Service Connect](../docs/manifest/lb-web-service.md#network-connect) enabled.
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
For example, imagine you have an environment with two services, `users` and `orders`, who both have [Service Connect](../docs/manifest/lb-web-service.md#network-connect) enabled.
```

File: site/content/blogs/release-v132.en.md [hunk 5]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
...
+Additionally, `foo` has an [RDS addon](../docs/developing/addons/workload.md) deployed that it uses for storing data.
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
Additionally, `orders` has an [RDS addon](../docs/developing/addons/workload.md) deployed that it uses for storing data.
```

File: site/content/blogs/release-v132.en.md [hunk 6]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
...
+To test `foo` locally, you can now run `copilot run local --proxy --name foo`, and your local `foo` containers can reach the following:
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
To test `orders` locally, you can now run `copilot run local --proxy --name foo`, and your local `foo` containers can reach the following:
```

File: site/content/blogs/release-v132.en.md [hunk 7]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
...
+
+- The `bar` service, using its service connect URL (which defaults to `http://bar:<port>`)
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
- The `users` service, using its service connect URL (which defaults to `http://bar:<port>`)
```

File: site/content/blogs/release-v132.en.md [hunk 8]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
...
+- `foo`'s RDS addon, using any DB instance url (like `app-env-svc-random-characters.us-west-2.rds.amazonaws.com:5432`) or DB cluster URL.
```
Comment (Lou1415926) [inline site/content/blogs/release-v132.en.md]:
```suggestion
- `orders`'s RDS database, using any DB instance url (like `app-env-svc-random-characters.us-west-2.rds.amazonaws.com:5432`) or DB cluster URL.
```

File: site/content/blogs/release-v132.en.md [hunk 9]
```diff
@@ -18,7 +18,7 @@ Thanks to every one of you who shows love and support for AWS Copilot.
 
 Copilot v1.32 brings big enhancements to help you develop more flexibly and efficiently:
 
-- **`copilot run local --proxy`**:
+- **`copilot run local --proxy`**: Proxy outbound traffic from your local containers to services and RDS instances in your environment VPC. [See detailed section](#copilot-run-local---proxy)
```
Comment (iamhopaul123) [inline site/content/blogs/release-v132.en.md]: Should we leave VPC out and use Copilot environment instead? As i would assume the traffic is still controlled by the env sg which means two env cannot talk to each other even if they share the same VPC by default.

**Review by Lou1415926 — APPROVED** [review]
🚢

File: site/content/blogs/release-v132.en.md [hunk 10]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
 
     See the section [Overview](../docs/concepts/overview.en.md) for a more detailed introduction to AWS Copilot.
 
-## 
+## `copilot run local --proxy`
+
+The new `--proxy` flag on `copilot run local` enhances your local development experience by making it possible for your local containers to talk to services deployed in your environment's VPC.
+
+For example, imagine you have an environment with two services, `users` and `orders`, that both have [Service Connect](../docs/manifest/lb-web-service.md#network-connect) enabled.
+Additionally, `orders` has an [RDS addon](../docs/developing/addons/workload.md) deployed that it uses for storing data.
```
Comment (iamhopaul123) [inline site/content/blogs/release-v132.en.md:40]:
```suggestion
For example, imagine you have an environment with two services, `users` and `orders`, that both have [Service Connect](../docs/manifest/lb-web-service.en.md#network-connect) enabled.
Additionally, `orders` has an [RDS addon](../docs/developing/addons/workload.en.md) deployed that it uses for storing data.
```

File: site/content/blogs/release-v132.en.md [hunk 11]
```diff
@@ -32,7 +32,16 @@ Copilot v1.32 brings big enhancements to help you develop more flexibly and effi
...
+To test `orders` locally, you can now run `copilot run local --proxy --name orders`, and your local `orders` containers can reach the following:
```
Comment (iamhopaul123) [inline site/content/blogs/release-v132.en.md]: Maybe
```suggestion
You can now run `copilot run local --proxy --name orders` to test the following connection for your local `orders` containers:
```

**Review by iamhopaul123 — APPROVED** [review]

**Pull request merged**